hphp/runtime/vm/jit/unique-stubs.cpp

```diff
@@ -1073,13 +1073,18 @@ TCA emitDecRefGeneric(CodeBlock& cb, DataBlock& data) {
 
 TCA emitEnterTCExit(CodeBlock& cb, DataBlock& data, UniqueStubs& us) {
   return vwrap(cb, data, [&] (Vout& v) {
+    bool saveFrame = true;
+
     // Eagerly save VM regs and realign the native stack.
     storeVMRegs(v);
 
     // Realign the native stack.
     switch (arch()) {
```

Comment:
@mxw I could not avoid this change in this unique stub, as PPC64 does not create a frame when entering the VM, there is no need to save the FP, so I added this switch.